src/components/voltage-level-choice.js

```diff
@@ -57,6 +62,25 @@ const VoltageLevelChoice = ({
 }) => {
     const classes = useStyles();
     const useName = useSelector((state) => state[PARAM_USE_NAME]);
+    const displayMode = useSelector((state) => state[PARAM_MAP_TREE_DISPLAY]);
```

Comment:
Ok but does this component which knows about drawers and their status (openned closed) and all ? Maybe you should move this computation of the position in `NetworkMapTab` component instead ?

Reply:
Keep this component simple. It only needs a position Object which give `top` and a `left` value and that's all.

Reply:
Yes that's right. I moved this code to NetworkMapTap